celestia.cfg

```diff
@@ -82,34 +82,34 @@ StarTextures
 {
 	# This texture will be used for any spectral type not listed
 	# in this block.
-	Default "astar.jpg"
+	Default "star-type-a.*"
 
-	O "bstar.*"
-	B "bstar.*"
-	A "astar.*"
-	F "astar.*"
-	G "gstar.*"
-	K "gstar.*"
-	M "mstar.*"
+	O "star-type-o.*"
+	B "star-type-b.*"
+	A "star-type-a.*"
+	F "star-type-f.*"
+	G "star-type-g.*"
+	K "star-type-k.*"
+	M "star-type-m.*"
 
 	# carbon stars
-	C "mstar.*"
-	R "mstar.*"  # former subclass of carbon star
-	N "mstar.*"  # former subclass of carbon star
-	S "mstar.*"  # roughly between M and C
+	C "star-type-m.*"
+	R "star-type-m.*"  # former subclass of carbon star
+	N "star-type-m.*"  # former subclass of carbon star
+	S "star-type-m.*"  # roughly between M and C
 
 	# Wolf-Rayet stars
-	WC "bstar.*"
-	WN "bstar.*"
+	WC "star-type-o.*"
+	WN "star-type-o.*"
 
 	# brown dwarfs
-	L "browndwarf.*"
-	T "browndwarf.*"
-	Y "browndwarf.*"
+	L "star-type-l.*"
+	T "star-type-t.*"
+	Y "star-type-t.*"
 
 	# stellar remnants
-	WD "astar.jpg"
-	NeutronStar "astar.jpg"
+	WD "star-type-d.*"
+	NeutronStar "NoTexture.*"
```

Comment:
What does this mean? There is no file called `NoTexture` in the other PR.

Reply:
Exactly, it's supposed to be a nonexistent texture. This is necessary because the neutron star texture is also used for black holes.

Reply:
bad idea. it's better to open an issue to fix BD texture

Reply:
#662

Reply:
Black hole texture issue resolved, neutron star texture changed to one that exists.